app/components/LongCode.tsx

```diff
@@ -5,8 +5,8 @@ import { Box } from '@material-ui/core';
 
 import { GOLD_DARK, GOLD_LIGHT } from '../constants/colors';
 
-const LUCKY_ARRAY_INDEX = [1, 9, 89, 97];
-const SUPER_LUCKY_ARRAY_INDEX = [0, 10, 88, 98];
+const LUCKY_ARRAY_INDEX = [1, 2, 104, 105];
+const SUPER_LUCKY_ARRAY_INDEX = [0, 3, 103, 106];
```

Comment:
The code's length is not guaranteed. This won't work. Instead, you should change the logic around make...
GOLD_LIGHT --> 2nd, 3rd, 3rd-to-last, 2nd-to-last
GOLD_DARK --> 1st, 4th, 4th-to-last, last